modelina-cli/test/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig",
+  "compilerOptions": {
+    "noEmit": true
+  },
+  "references": [
```

Comment:
Do we need this now? Since now we have been using modelina as a dependency instead of importing from parent directory.

Reply:
You should not need this no 🙂